docs/source/_static/data_formatting_examples/exec_format_structured_data.py

```diff
@@ -0,0 +1,151 @@
+"""
+Format Structured Data
+======================
+
+Format a dataset where the filenames of the tiles provide most of the metadata required to
+organize them into the 5D set (image_type, fov, round, ch, zplane) of 2D tiles  (y, x).  The
+remaining metadata, namely physical coordinates of the tiles, are provided in a CSV file.
+
+The stem of the tiles' filenames must be
+`<image_type>-f<fov_id>-r<round_label>-c<ch_label>-z<zplane_label>`.  The extension should be one
+of the supported tile formats (:py:class:`~slicedimage.ImageFormat`).  For example, the file
+`nuclei-f0-r2-c3-z33.tiff` would belong to the nuclei image, fov 0, round 2, channel 3, zplane 33.
+
+The CSV file must have a header, and must contain the following columns:
+
+====== ======== ====================================================================================
+Column Required Notes
+------ -------- ------------------------------------------------------------------------------------
+fov    yes
+round  yes
```

Comment:
I'm not sure if the code accounts for this, but Matlab is 1 based indexing, vs. Python 0 based indexing. If a Matlab user uses 1 to mean 0, will your image stack creation still work as expected? If not, we should be clear in documentation that we require 0 based indexing.

Reply:
sorry, i should add that i've noticed Matlab users write out file names where round = 1-4 as opposed to 0-3 and I've had to add -1s in my code to account for this when making the correctly shape numpy/xarrays

Reply:
This is a great observation. 💯

Reply:
Well, what happens is that we write the literal r/ch/z, so it will work.  If you index into the xarray using r=0 when you write out r={1,2,3}, then you will not be a happy camper.  However, if you index into the array using the exact values you wrote out, things will work.